### DISCUSSION TITLE
Google Git PURL type

### DISCUSSION THREAD

**Discussion by jsdratm** [package-url/purl-spec#336] [Q&A]
I ran across this software component and I'm having trouble determining a PURL for it:

https://chromium.googlesource.com/libyuv/libyuv/

Any thoughts?

**Comment by jkowalleck** [comment]
how about `pkg:generic/libyuv?vcs_url=https://chromium.googlesource.com/libyuv/libyuv` ?

**Comment by pombredanne** [comment]
I second @jkowalleck suggestion for now.  We should also refine and evolve generic types, possibly using domain names as an adjunct, and/or considering publishing a open registry of known generic PURL especially for C/C++ packages, or even a new type for these, but here https://chromium.googlesource.com seems to strictly be a git host.

**Comment by pombredanne** [comment]
Making this a discussion, in case there is more to it than the eye can see!
